valgrind.suppress

```diff
@@ -171,6 +171,20 @@
    fun:ngx_create_pool
    fun:main
 }
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds: definite
+   fun:malloc
+   fun:ngx_alloc
+   fun:ngx_event_process_init
```

Comment:
IMHO, we already have this suppression: https://github.com/openresty/lua-nginx-module/blob/master/valgrind.suppress#L94

Reply:
Didn't notice that. Thanks for pointing out. Closing now.